test/models/poem_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```

Comment:
We will be using Rspec so u can neglect this files.

Whatever from test/* u can remove